Jenkinsfile

```diff
@@ -157,7 +157,10 @@ pipeline {
                 withCredentials([ossrhCredentials]) {
                     sh """\
                         #!/usr/bin/env bash
-                        mvn clean install $BUILD_ARGS $EXTRA_BUILD_ARGS -s .jenkins/settings.xml
+                        set -xe
```

Comment:
It is in case of jenkins issues, the good practice is always to put xe in case of bash issue, we well then directly have access to the **$BUILD_ARGS** and **$extraBuildParams** contents.
We will then be quicket to understand the problem. 
With bash and the space and escaping management it help a lot.